docker/src/main/java/brooklyn/entity/container/docker/DockerInfrastructureImpl.java

```diff
@@ -101,6 +102,8 @@
 
     private transient Object mutex = new Object[0];
 
+    private AtomicInteger infrastructureID = new AtomicInteger();
```

Comment:
I think this should be `static final` so that it is shared across all instances,?